Client/game_sa/CBuildingsPoolSA.cpp

```diff
@@ -48,6 +48,16 @@ inline bool CBuildingsPoolSA::AddBuildingToPool(CClientBuilding* pClientBuilding
     return true;
 }
 
+CClientEntity* CBuildingsPoolSA::GetClientBuilding(CBuildingSAInterface* pGameInterface) const noexcept
+{
+    auto poolIndex = (*m_ppBuildingPoolInterface)->GetObjectIndex(pGameInterface);
```

Comment:
It's better to avoid `auto` for small types like int.

Reply:
agree